networkx/algorithms/dag.py

```diff
@@ -287,3 +289,75 @@ def is_aperiodic(G):
         return g==1
     else:
         return g==1 and nx.is_aperiodic(G.subgraph(set(G)-set(levels)))
+
+def transitive_closure(G):
+    """ Based on http://www.ics.uci.edu/~eppstein/PADS/PartialOrder.py
```

Comment:
This should go to references section. The first sentence should always be a short description of what the function does.